TFT/src/User/API/parseACK.c

```diff
@@ -1223,7 +1223,7 @@ void parseACK(void)
       {
         infoMachineSettings.buildPercent = ack_value();
       }
-      else if (ack_seen("Cap:CHAMBER_TEMPERATURE:"))
+      else if (ack_seen("Cap:CHAMBER_TEMPERATURE:") && infoSettings.chamber_en == DISABLED)  // auto-detect only if set to disabled
```

Comment:
This is kind of counterintuitive. In English "disable" means to turn something that was functional into an inoperative state. 
I speak only for myself, but I surely do not want something that I disabled to be functional.